swap_meet/item.py

```diff
@@ -1,2 +1,23 @@
+
 class Item:
-    pass
\ No newline at end of file
+    category = ""
+    
+    def __init__(self, category = category, condition = 0):
+        self.category = category
+        self.condition = condition
+
+    def __str__(self):
+        self = "Hello World!"
+        return f"{self}"
```

Comment:
We shouldn't re-assign `self` within an instance method. Remember that `self` is used to pass in a reference to the instance a method was called on. Re-assigning `self` won't affect that instance, but if we _did_ need to get a value from it, this would make that impossible. Here, we can simply return the desired string directly.

```python
    def __str__(self):
        return "Hello World!"
```